<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注入成功</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pB_download">
        <property name="text">
         <string>下载注入后的修复模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_download_2">
        <property name="text">
         <string>下载注入后原模型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_refine">
        <property name="text">
         <string>回到上一个窗口重新调整参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_back">
        <property name="text">
         <string>回到主窗口继续修复其他模型</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
